modules/logging-in-by-using-the-web-console.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// *installing/validating-an-installation.adoc
+
+[id="logging-in-by-using-the-web-console_{context}"]
+= Logging in to the cluster by using the web console
+
+You can log into your cluster as `kubeadmin` by using the {product-title} web console after the Console Operator becomes available.
+
+.Prerequisites
+
+* You have access to the installation host.
+* The Console Operator is available on your cluster.
+
+.Procedure
+
+. Obtain the `kubeadmin` password from the `kubeadmin-password` file on the installation host:
```

Comment:
`kubeadmin` user?

Reply:
Thank you for highlighting this. I have subsequently changed this line to:

. Obtain the password for the `kubeadmin` user from the `kubeadmin-password` file on the installation host: